commit 3b54f52ecad6a8e13623358979ea3d236f80d8fa
Author: Trent Mick <trentm@gmail.com>
Date:   2019-05-28T16:07:45-07:00 (4 months ago)
    
    joyent/node-triton#169 `triton rbac ...` broken by #108 (bump ver to 7.1.1)

diff --git a/CHANGES.md b/CHANGES.md
index 5998b04..de8ed0c 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -6,6 +6,11 @@ Known issues:
 
 ## not yet released
 
+## 7.1.1
+
+- [joyent/node-triton#169] Fix `triton rbac ...` commands that were
+  all broken by #108 in v5 (by github.com/YangYong3).
+
 ## 7.1.0
 
 - [TRITON-967] Add support for listing, adding, deleting, and resizing
diff --git a/package.json b/package.json
index 3e545ab..8dc4921 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "triton",
   "description": "Joyent Triton CLI and client (https://www.joyent.com/triton)",
-  "version": "7.1.0",
+  "version": "7.1.1",
   "author": "Joyent (joyent.com)",
   "homepage": "https://github.com/joyent/node-triton",
   "dependencies": {
